source/learners/sfd_discussions/find_follow_conversations.rst

```diff
@@ -0,0 +1,155 @@
+.. _find-follow-conversations:
+
+##############################################
+Finding and Following the Right Conversations
+##############################################
+
+Discussions can grow quickly as a course progresses. 
+Knowing how to browse and search helps you save time and discover conversations that matter to you. 
+
+.. contents::
+  :local:
+  :depth: 1
+
+.. _Navigating the Discussions page:
+
+********************************
+Navigating the Discussions page
+********************************
+
+When you open the Discussions tab in your course, you see four tabs at the top, as shown in the image below. 
```

Comment:
Add the image right here, before the bullets. It is clearer.

Reply:
Done